static/app/components/groupPreviewTooltip/index.tsx

```diff
@@ -18,36 +22,49 @@ function GroupPreviewTooltip({
   groupId,
   groupingCurrentLevel,
   issueCategory,
+  issueType,
+  project,
   query,
 }: GroupPreviewTooltipProps) {
   if (!issueCategory) {
     return null;
   }
 
-  switch (issueCategory) {
-    case IssueCategory.ERROR:
-      return (
-        <StackTracePreview
-          groupId={groupId}
-          groupingCurrentLevel={groupingCurrentLevel}
-          query={query}
-        >
-          {children}
-        </StackTracePreview>
-      );
-    case IssueCategory.PERFORMANCE:
-      return (
-        <SpanEvidencePreview groupId={groupId} query={query}>
-          {children}
-        </SpanEvidencePreview>
-      );
-    default:
-      return (
-        <EvidencePreview groupId={groupId} query={query}>
-          {children}
-        </EvidencePreview>
-      );
+  const issueTypeConfig = project
+    ? getConfigForIssueType(
+        {
+          issueCategory,
+          issueType,
+        },
+        project
+      )
+    : null;
+
+  if (issueTypeConfig?.spanEvidence.enabled) {
+    return (
+      <SpanEvidencePreview groupId={groupId} query={query}>
+        {children}
+      </SpanEvidencePreview>
+    );
+  }
+
+  if (issueCategory === IssueCategory.ERROR) {
```

Comment:
is this IssueCategory check one of the unavoidable ones?

Reply:
Good call, I can use the config here too. Fixed